### ISSUE TITLE
testing: mint unit tests consistently fail

### ISSUE THREAD

**Issue by jharveyb** [lightninglabs/taproot-assets#974]
From #971 

```bash
--- FAIL: TestBatchedAssetIssuance (11.88s)
    test_sqlite.go:11: Creating new SQLite DB handle for testing: /tmp/TestBatchedAssetIssuance3571172735/001/tmp.db
    --- FAIL: TestBatchedAssetIssuance/basic_asset_creation (1.60s)
        planter_test.go:959: 
            	Error Trace:	/home/runner/work/taproot-assets/taproot-assets/tapgarden/planter_test.go:959
            	            				/home/runner/work/taproot-assets/taproot-assets/tapgarden/planter_test.go:1097
            	            				/home/runner/work/taproot-assets/taproot-assets/tapgarden/planter_test.go:1829
            	Error:      	Received unexpected error:
            	            	got empty confirmation event in batch
            	Test:       	TestBatchedAssetIssuance/basic_asset_creation
    test_sqlite.go:11: Creating new SQLite DB handle for testing: /tmp/TestBatchedAssetIssuance3571172735/002/tmp.db
    test_sqlite.go:11: Creating new SQLite DB handle for testing: /tmp/TestBatchedAssetIssuance3571172735/003/tmp.db
    test_sqlite.go:11: Creating new SQLite DB handle for testing: /tmp/TestBatchedAssetIssuance3571172735/004/tmp.db
    test_sqlite.go:11: Creating new SQLite DB handle for testing: /tmp/TestBatchedAssetIssuance3571172735/005/tmp.db
    test_sqlite.go:11: Creating new SQLite DB handle for testing: /tmp/TestBatchedAssetIssuance3571172735/006/tmp.db
FAIL
FAIL	github.com/lightninglabs/taproot-assets/tapgarden	17.647s
FAIL
```

**Comment by jharveyb** [comment]
Seems to be showing up as a consistent failure now, not a flake.

**Comment by jharveyb** [comment]
Seems to be showing up as a consistent failure now, not a flake. Current theory is as an issue with setting `MockChainBridge.errConf`.

**Comment by jharveyb** [comment]
Closed by #1031 for now.
